Safe-DS/Library-Analyzer
# feat: extracting valid literals from documentation

Closes #46.

### Summary of Changes

Currently, valid values for the enum annotations were determined only via the type string. In order to achieve a higher recognition rate of the enum annotations, the recognition of the valid values was extended to the description.


### Testing Instructions

1. Run `pytest` for `test_extract_valid_literals.py`.
2. Check the results of `pytest`.
